config/initializers/rack_cors.rb

```diff
@@ -45,7 +45,7 @@
   }
 
   allow do
-    origins { _1.ends_with?(Keygen::DOMAIN) }
+    origins Keygen::Portal::ORIGIN
```

Comment:
Similar vulnerability that triggered this change: https://github.com/zeromicro/go-zero/security/advisories/GHSA-fgxv-gw55-r5fq